templates/app/client/components/socket(socketio)/socket.service.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
-
+import _ from 'lodash';
```

Comment:
Have previously needed to use `import * as _ from 'lodash';` to avoid getting an error in typescript, should probably check this out.